CHANGELOG.md

```diff
@@ -3,10 +3,13 @@
 ### Features
 
 ### Fixes
+  
+### Chore & Maintenance
   - Fixed bug where required input field was not recognizing the character "s" as a valid input because of Regex issues. 
```

Comment:
shouldn't the bug fix go under fixes instead of chore and maintenance?

Reply:
good catch. Fixed